### ISSUE TITLE
addition of feedback form 

### ISSUE THREAD

**Issue by vedanshipathak** [ashutoshgithubs/iemhacks_DEV-G#72]
@ashutoshgithubs  currently the website is lacking a feedback form. i have an experience in react will be able to make a form .Kindly assign this issue under GSSOC'24

**Comment by ashutoshgithubs** [comment]
Hi @vedanshipathak , I think **Facing an isssue** button can be modified somehow to serve as both functionality. Because form for both **FEEDBACK** and **ISSSUE** remain same. So think accordingly to avoid duplicacy and let me know. You suggestion is welcome.

**Comment by vedanshipathak** [comment]
@ashutoshgithubs Yes it can we modified but according to my opinion facing issues should be resolved within time for eg. struggling opening certain content (for that a chat bot can be used ) and feedback form would be more like about the user experience how much they loved using this website including  the rating system . This would what i will suggest
Tell me if you differ on that .
Thankyou:)

**Comment by ashutoshgithubs** [comment]
> @ashutoshgithubs Yes it can we modified but according to my opinion facing issues should be resolved within time for eg. struggling opening certain content (for that a chat bot can be used ) and feedback form would be more like about the user experience how much they loved using this website including the rating system . This would what i will suggest Tell me if you differ on that . Thankyou:)

So are you planning to create any separate section for this?

**Comment by vedanshipathak** [comment]
@ashutoshgithubs i was planning to add in the footer section .i can create a modal for it

**Comment by nishant0708** [comment]
can you assign me this issue under gssoc 24.
